msal/src/Microsoft.Identity.Client/Features/DeviceCode/IPublicClientApplication.cs

```diff
@@ -1,102 +1,132 @@
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
 using System.Threading;
-using System.Threading.Tasks;
-
-namespace Microsoft.Identity.Client
-{
-    /// <summary>
-    /// Interface to be used with desktop or mobile applications (Desktop / UWP / Xamarin.iOS / Xamarin.Android).
-    /// public client applications are not trusted to safely keep application secrets, and therefore they only access Web APIs in the name of the user only 
-    /// (they only support public client flows). For details see https://aka.ms/msal-net-client-applications
-    /// </summary>
-    public partial interface IPublicClientApplication : IClientApplicationBase
-    {
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Interface to be used with desktop or mobile applications (Desktop / UWP / Xamarin.iOS / Xamarin.Android).
+    /// public client applications are not trusted to safely keep application secrets, and therefore they only access Web APIs in the name of the user only 
+    /// (they only support public client flows). For details see https://aka.ms/msal-net-client-applications
+    /// </summary>
+    public partial interface IPublicClientApplication : IClientApplicationBase
+    {
         /// <summary>
-        /// Acquires device code from the authority and returns it to the caller via
-        /// the deviceCodeResultCallback. The function then proceeds to poll for the security
-        /// token which is granted upon successful login based on the device code information.
+        /// Acquires a security token on a device without a Web browser, by letting the user authenticate on 
+        /// another device. This is done in two steps:
+        /// <list type="bullet">
+        /// <item><description>the method first acquires a device code from the authority and returns it to the caller via
+        /// the <paramref name="deviceCodeResultCallback"/>. This callback takes care of interacting with the user
+        /// to direct him to authenticate (to a specific URL, with a code)</description></item>

```

Comment:
nit:  should we have a gender-neutral stance on pronouns?  "to direct them" perhaps?

Reply:
thanks. Done!